client/resources/aria-at-test-io-format.mjs

```diff
@@ -1642,7 +1642,7 @@ const UnexpectedBehaviorImpactMap = createEnumMap({
 /**
  * Command result derived from priority 1 and 2 assertions.
  *
- * Support is "FAILING" is priority 1 assertions fail. Support is "ALL REQUIRED"
+ * Support is "FAILING" is priority 1 assertions fail. Support is "ALL MUST"
```

Comment:
I don't think this has to be changed

Reply:
Agreed. If you do change it, make sure to look at server/util/getMetrics.js which still uses the term "All Required." Probably better to just leave it as-is.